packages/main/src/TimePicker.ts

```diff
@@ -457,8 +467,8 @@ class TimePicker extends UI5Element implements IFormInputElement {
 	 * @returns Resolves when the Inputs popover is closed
 	 */
 	closeInputsPopover() {
-		const popover = this._getInputsPopover();
-		popover.open = false;
+		const popover = this._inputsPopover;
```

Comment:
There is already a `closeInputsPopover` function that does this, please reuse it here.